docker-vertica/Dockerfile

```diff
@@ -160,7 +160,9 @@ RUN set -x \
   && chmod u+s /usr/sbin/cron \
   # Untar the init program that was downloaded earlier
   && tar -C / -Jxpf /tmp/s6-overlay-x86_64.tar.xz \
-  && tar -C / -Jxpf /tmp/s6-overlay-noarch.tar.xz
+  && tar -C / -Jxpf /tmp/s6-overlay-noarch.tar.xz \
+  # delete old host keys
+  && rm -rf /etc/ssh/ssh_host* 
```

Comment:
Where are these host keys being added? I would have thought that removal of `ssh-keygen` would have been enough.

Reply:
I observed that some keys' creation time corresponded to the build time and others to pod creation time, so that is why I removed all the keys before `ssh-keygen`.